samples/Calling/src/app/views/HomeScreen.tsx

```diff
@@ -257,6 +266,18 @@ export const HomeScreen = (props: HomeScreenProps): JSX.Element => {
                 </Stack>
               )
             }
+            {
+              /* @conditional-compile-remove(teams-adhoc-call) */ teamsAdhocChosen && (
+                <Stack>
+                  <TextField
+                    className={outboundTextField}
+                    label={"Teams ID's"}
```

Comment:
Can we say "Teams user ID's" and maybe in the placeholder give an example since we need to prefixes to have the handlers parse correctly? or alternatively give a callout like with the alternate callerId showing an example of a good user ID to avoid confusion when others want to try it?

Reply:
Good point. Changed label and added a callout.